modules/standard/Math.chpl

```diff
@@ -888,6 +888,87 @@ module Math {
     return logBasePow2(val, 1);
   }
 
+  //
+  // min and max
+  //
+
+  /* Returns the maximum value of two integer arguments.
```

Comment:
This feels a little crowded, especially in comparison to the previous documentation.  I would be inclined to reduce it to just the generic, two argument version and the three argument version, and "no doc" the rest, moving the "Math.NAN" information on the two argument version as a white lie.

Reply:
I left the generic two-args, any args, and param versions. Is that acceptable?